airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java

```diff
@@ -49,6 +49,8 @@ public class SyncWorkflowImpl implements SyncWorkflow {
   private static final int NORMALIZATION_SUMMARY_CHECK_CURRENT_VERSION = 1;
   private static final String AUTO_DETECT_SCHEMA_TAG = "auto_detect_schema";
   private static final int AUTO_DETECT_SCHEMA_VERSION = 2;
+  private static final String REMOVE_ACTIVITY_FOR_NORM_INPUT = "remove_activity_for_norm_input";
```

Comment:
this version name is a bit confusing now that you're no longer deciding to remove the activity. Maybe `use_minimal_norm_input` is a better version name now?